src/constants.ts

```diff
@@ -7,10 +7,6 @@ export const LineFeed = '\n';
 export const CarriageReturnLineFeedCode = 0;
 export const LineFeedCode = 1;
 
-export const ScriptTargetES2015 = 2;
```

Comment:
> Also @arcanis - would this create issues for yarn PnP support?

 Not if `typescript` is defined as either `dependencies` or `peerDependencies` (`devDependencies` won't do since they won't be considered part of the dependency tree when installed by consumers).

Reply:
Cool - it's already a `peerDependency` so I guess we're good https://github.com/TypeStrong/ts-loader/blob/48626a99ecee68875df491a75f7137543a569db5/package.json#L94

```
  "peerDependencies": {
    "typescript": "*"
  }
```